src/tools/illink/test/Mono.Linker.Tests.Shared/Mono.Linker.Tests.Shared.projitems

```diff
@@ -0,0 +1,5 @@
+<Project>
```

Comment:
Personally I would rename this to something like `Trim.Tests.Shared.projitems` (and the directory and so on).
Eventually we should also rename `ILLink.Shared` to something else - maybe `Trim.Shared`, or `Trimming.Shared`. It's weird to have ILLink in that name for several reasons :-).